src/votypes.jl

```diff
@@ -92,7 +92,13 @@ _parse(::Type{Vector{T}}, s) where {T} = map(x -> _parse(T, x), split(s))
 
 _parse_binary(::Type{String}, data) = String(copy(data))
 _parse_binary(::Type{Char}, data) = Char(only(data))
-_parse_binary(::Type{T}, data) where {T<:Union{Bool,Int,Int32,Int16,Float32,Float64,ComplexF32,ComplexF64}} = reinterpret(T, data) |> only |> bswap
+_parse_binary(::Type{T}, data) where {T<:Union{Int,Int32,Int16,Float32,Float64,ComplexF32,ComplexF64}} = reinterpret(T, data) |> only |> bswap
+function _parse_binary(::Type{Bool}, data)
+    c = data[1] |> Char
+    c in ('T', 't', '1') && return true
+    c in ('F', 'f', '0') && return false
+    missing
```

Comment:
This is in part a casualty of breaking up the original PR into smaller ones - that `missing` is not in most cases necessary for `BINARY2` serialization but it is for `BINARY`.  It is at least harmless for BINARY2, and covers the case where the table has indicated a NULL binary value with an 0x00, 0x20 or 0x3f rather than a bit in the NULL flags array, which is legal (though arguably eccentric in `BINARY2`) for scalars, and necessary for boolean array elements.

[Disclaimer: I don't understand how that function can work without a fall-through value following the T/F cases, but without one it returns `false` for input values of `" "` or `"?"`, which is wrong]

Reply:
Oh I see... clever, but not very transparent (to me, anyway)